js/src/features/project/components/Team/TeamMembersListItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { List, Tooltip, Select } from "antd";
+import { CloseCircleOutlined } from "@ant-design/icons";
+
+import { ProjectTeamMember, Role } from "ProjectModels";
+
+interface Props {
+  name: string;
+  roles: Role[];
+  member: ProjectTeamMember;
+}
+
+const { Option } = Select;
+
+export const TeamMembersListItem: React.FC<Props> = ({
+  name,
+  roles,
+  member,
+}) => {
+  const actions = [
+    <Tooltip title="Dismiss">
+      <CloseCircleOutlined />
+    </Tooltip>,
+  ];
+
+  function getRole() {
+    return roles.find((role) => role.id === member.role)?.name;
+  }
+
+  return (
+    <List.Item actions={actions}>
+      <List.Item.Meta title={name} />
+      <Select defaultValue={getRole()} style={{ width: 120 }}>
```

Comment:
Why do you use a function here? Can we replace it with variable?

Reply:
fix